src/Simplex/Messaging/Agent/TRcvQueues.hs

```diff
@@ -113,7 +113,7 @@ isSession rq (uId, srv, connId_) =
 
 instance Queue RcvQueue where
   connId' = connId
-  qKey rq = (userId rq, server rq, connId rq)
+  qKey rq = (userId rq, server rq, rcvId rq)
```

Comment:
This was a proper bug that could have resulted in incorrect subscriptions, particularly when switching connections - both queues would be added under the same key.